### ISSUE TITLE
Cloning Gdrive Folders BUG

### ISSUE THREAD

**Issue by askfriends** [viperadnan-git/google-drive-telegram-bot#91]
hi;
I dont know if this is active or not bcoz there are many issues out there and OP is not responding, i hope he will respond to everyone and fix all bugs;

I am copying gdrive link which has many sub folders, but it copy all the folders in the destination separately, the actual main folder in which subfolders are available is empty, and all subfolders are copied in parent folder separately. can u fix this?

**Comment by mridun-gupta** [comment]
Yes, sir I am also facing the same issue do you get any solution

**Comment by viperadnan-git** [comment]
Actually i fixed the issue but it's not updated in repository yet

**Comment by askfriends** [comment]
> Actually i fixed the issue but it's not updated in repository yet

Please release the update bcoz thats a must feature.

**Comment by mridun-gupta** [comment]
> > Actually i fixed the issue but it's not updated in repository yet
> 
> Please release the update bcoz thats a must feature.

yes

**Comment by Bodrul98** [comment]
has the issue fixed?

**Comment by mridun-gupta** [comment]
> has the issue fixed?

may be not yet @viperadnan-git

**Comment by viperadnan-git** [comment]
Fixed in [gdnan](https://pypi.org/project/gdnan/) but it's need to be ported in this repo, i am thinking to use rclone instead.

**Comment by mridun-gupta** [comment]
oh! i am eager for it! kindly notify after which they issues gets fixed

**Comment by askfriends** [comment]
> Fixed in [gdnan](https://pypi.org/project/gdnan/) but it's need to be ported in this repo, i am thinking to use rclone instead.

bro why dont u release update in this repository?
this will help everyone using this bot.
